src/components/shared/Heading/H3.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import Heading, { HeadingAppearance, HeadingProps } from './Heading';
+
+const displayName = 'Heading.H3';
+
+export interface H3Props extends Omit<HeadingProps, 'appearance'> {
+  appearance?: Partial<HeadingAppearance>;
+}
+
+const H3 = ({ appearance, ...props }: H3Props) => (
```

Comment:
I don't like the naming here, especially since the `H3` custom component can be easily confused with `h3` the native element.

Please rename this to `Heading3` so that it's more obvious